chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/validation/reviewer/SvapReviewer.java

```diff
@@ -100,9 +100,15 @@ private void reviewRegulatoryTextCitationRequired(CertifiedProductSearchDetails
 
     private void reviewSvapMarkedAsReplaced(CertifiedProductSearchDetails listing,
             CertificationResult certResult, CertificationResultSvap svap) {
-        if (svap.getSvapId() != null && BooleanUtils.isTrue(svap.getReplaced())) {
-            listing.getWarningMessages().add(msgUtil.getMessage("listing.criteria.svap.replaced",
+        if (svap.getSvapId() != null
```

Comment:
There is a suite of svap AQA tests for flexible upload. We hadn't covered this particular message, but I tested the file on DEV and noticed that this error now appears for one of the uploaded listings being tested - so this PR will need an AQA update. I am comfortable doing it since I wrote most of those if you'd like. Or happy to show you.